src/main/java/org/kohsuke/github/GHEventPayload.java

```diff
@@ -121,16 +140,339 @@ void wrapUp(GitHub root) {
     }
 
     /**
+     * A comment was added to a commit
+     *
+     * @see <a href="http://developer.github.com/v3/activity/events/types/#commitcommentevent">authoritative source</a>
+     */
+    @SuppressFBWarnings(value = {"UWF_FIELD_NOT_INITIALIZED_IN_CONSTRUCTOR", "NP_UNWRITTEN_FIELD" },
+            justification = "Constructed by JSON deserialization")
+    public static class CommitComment extends GHEventPayload {
```

Comment:
it would have been better if the design was originally that way... but needs to be in same package due to current API design (FTR I would not have designed this API this way as it is very tricky to unpick the stuff that requires a connection from the stuff that doesn't... but we are where we are)